kubernetes/Makefile

```diff
@@ -8,6 +8,7 @@ cluster:
 		k3d cluster create \
 		--port="80:80@loadbalancer" \
 		--port="443:443@loadbalancer" \
+		--port="3000:3000@loadbalancer" \
```

Comment:
```suggestion
```

We don't want to open the port: instead, we want an ingress rule that would allow the URL `http://localhost/grafana` to serve the grafana content.